src/main/scala/no/gdl/bookapi/repository/TranslationRepository.scala

```diff
@@ -19,6 +19,8 @@ trait TranslationRepository {
   this: ContributorRepository =>
   val translationRepository: TranslationRepository
 
+  val countAllBeforeLimiting = sqls"count(*) over ()"
```

Comment:
Scalikejdbc doesn't provide this, but we need it when counting and selecting (with limiting and offset) at the same time.